lib/modifications.rb

```diff
@@ -504,5 +662,63 @@ def self.create_bundle_entry(resource)
         entry.request.url = resource.resourceType
         entry
     end
+
+    def self.get_reference_type(b, reference_string)
+      id = reference_string.split(':').last
```

Comment:
Splitting on `:` because they are `urn:uid:...`?

Reply:
Yeah, coming out of Synthea, all the references are `urn:uuid:<whatever the ID is>`, so we have to split on `:` to get the actual ID.